atlas/atlas.sh

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+readonly ATLAS_VERSION='1.1.0'
+readonly ATLAS_FILE="apache-atlas-${ATLAS_VERSION}-bin"
+readonly ATLAS_GCS_URL="gs://dariusz-init-actions/atlas/${ATLAS_FILE}.tar.gz" # TODO(Aniszewski): change to official GCS location before merging
+readonly ATLAS_HOME='/etc/atlas'
+readonly ATLAS_CONFIG="${ATLAS_HOME}/conf/atlas-application.properties"
+readonly ATLAS_TMP_TARBALL_PATH='/tmp/atlas-${ATLAS_VERSION}.tar.gz'
+readonly INIT_SCRIPT='/usr/lib/systemd/system/atlas.service'
+readonly ATLAS_ADMIN_USERNAME="$(/usr/share/google/get_metadata_value attributes/ATLAS_ADMIN_USERNAME)"
+readonly ATLAS_ADMIN_PASSWORD_SHA256="$(/usr/share/google/get_metadata_value attributes/ATLAS_ADMIN_PASSWORD_SHA256)"
+readonly MASTER=$(/usr/share/google/get_metadata_value attributes/dataproc-master)
+readonly ADDITIONAL_MASTER=$(/usr/share/google/get_metadata_value attributes/dataproc-master-additional)
+
+
+function err() {
+  echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $@" >&2
+  return 1
+}
+
+
+function check_prerequisites(){
+  # check for Zookeeper
+  echo stat | nc localhost 2181 || err 'Zookeeper not found'
+
+  # check for HBase
+  echo "list" | hbase shell || err 'HBase not found'
+
+  # check for Solr
+  curl 'http://localhost:8983/solr' || err 'Solr not found'
+
+  if [[ -n "${ADDITIONAL_MASTER}" ]]; then
+    # check for Kafka on HA
+    ls /usr/lib/kafka &>/dev/null || err 'Kafka not found'
+  fi
+}
+
+function download_atlas(){
+  gsutil cp "${ATLAS_GCS_URL}" "${ATLAS_TMP_TARBALL_PATH}"
+  tar -xf "${ATLAS_TMP_TARBALL_PATH}"
+  rm "${ATLAS_TMP_TARBALL_PATH}"
+  mv "apache-atlas-${ATLAS_VERSION}" "${ATLAS_HOME}"
+}
+
+function configure_solr(){
+  if [[ $(hostname) == "${MASTER}" ]]; then
+    # configure Solr only on the one actual Master node
+    runuser -l solr -s /bin/bash -c "/usr/lib/solr/bin/solr create -c vertex_index -d /etc/atlas/conf/solr -shards 3"
```

Comment:
If it is only 1 node cluster, you can use 1 shard right?

Reply:
@roderickyao is this comment still relevant?

Reply:
no. we can close this.